jinja/node.go

```diff
@@ -0,0 +1,50 @@
+package jinja
+
+import "fmt"
+
+type NodeInter interface {
+	String() string
+}
+
+type Node struct {
+	tokens   []Token
+	complete bool
+}
+
+func (n Node) isComplete() bool {
+	return n.complete
+}
+
+func (n Node) String() string {
+	s := ""
+	for _, t := range n.tokens {
+		s += fmt.Sprintf("'%+v'", t.String())
+	}
+	return s
+}
+
+type ExpressionNode struct {
```

Comment:
This is technically a BinaryNode because it's for a binary operator with a left and right child.